skietbaan_fe/src/components/RegisterStyles.css

```diff
@@ -50,22 +48,60 @@ input.invalid{
   color:red;
   font-size: 12px;
 }
+.header-container{
+  margin-left: -15px;
+    margin-right: -15px;
+    background-color: white;
+    padding-bottom: 5px;
+    padding-top: 5px;
+    text-align: center;
+
+}
+.front-white{
+  color:white
+}
+.button-container{
+  position: relative;
```

Comment:
inconsistent indentation and throughout the file

Reply:
fixed